communications/blogs/turing_joins_binder_federation.md

```diff
@@ -0,0 +1,102 @@
+
+## The Binder Federation
+
+[The Binder Federation](https://blog.jupyter.org/the-international-binder-federation-4f6235c1537e) is an experiment in open infrastructure for data science and beyond.
+It is a community of scientists and open-source developers dedicated to scientific reproducibility, communicating ideas through interactive computing, and providing real-time access to results and analyses.
+The Binder Federation provides interactive data analytics environments to roughly 10,000 unique repositories via 300,000 launches a month, and it relies on its team of partner BinderHubs to collaborate in providing this service.
+
+GKE, OVH, GESIS and now new members the Turing pool their resources to create a sustainable, open-source infrastructure that allows easy creation, sharing of and access to reproducible computational environments.
+Pooling resources in such a way is an important step for the resilience and sustanability of Project Binder since the dependency between mybinder.org and its funding is now spread across many sources.
+It also facilitates the decentralisation of the knowledge required to deploy and manage a BinderHub.
+
+_Turing Way_ contributor [Dr Sarah Gibson](https://www.turing.ac.uk/people/researchers/sarah-gibson) is also a [mybinder.org](https://mybinder.org) operator.
+The operators are responsible for the day-to-day operations and maintainance of the main website.
+She was therefore perfectly positioned to lead this project and bring the Turing into the Binder Federation.
+
+> I'm so excited that we've successfully merged our resources at the Turing into the mybinder.org infrastructure!
+> This achievement marks Binder's maturity as a federated project and the Turing's commitment to supporting open-source tooling and communities.
+>
+> -- Dr Sarah Gibson
+
+![Federation](https://miro.medium.com/max/1666/1*KU35naJhl1LSDxKY8gog4g.png)
+
+## How Can You Get Involved?
+
+If this has piqued your interest, there are a few ways to get involved in the Binder Federation!
+
+One way is to become a [mybinder.org](https://mybinder.org) operator.
+This is a low-tech contribution which involves hanging out in the project's [gitter room](https://gitter.im/jupyterhub/binder) answering questions, providing constructive input in the [GitHub repo](https://github.com/jupyterhub/binderhub), and being an active member of the community.
+Invitations to join the operating team are extended to outstanding members of our community.
+
+Alternatively, you could join the federation by facilitating the deployment of a  BinderHub!
+You can donate a cluster or cloud credits to the mybinder.org operating team.
+Your deployment would then be managed from the [mybinder.org-deploy repo](https://github.com/jupyterhub/mybinder.org-deploy) with input from the team and community.
+This is a more high-tech contribution but over a shorter time period.
```

Comment:
Related to the low-tech above: can we somehow rephrase this? Maybe we should address both at the same time when rephrasing.

When I started reading line 62 I thought this was going to be the section for people who are good at politics, making connections, networking, etc. Along the lines of "Do you like this cause and want to help out? Good at introducing people to each other and making money flow? This is how you can help!" but I think that isn't what you had in mind.

Reply:
This is @KirstieJane's idea of someone giving us a bunch of credits/money and we deploy another cluster as we see fit.